migrations/20201210150000-fix-host-payment-methods-and-transactions.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    const [, updateTs] = await queryInterface.sequelize.query(`
+      UPDATE "Transactions"
+      SET "amountInHostCurrency" = "amount", "hostCurrencyFxRate" = 1
+      FROM "PaymentMethods" pm, "Collectives" c
+      WHERE
+        "Transactions"."currency" = "Transactions"."hostCurrency"
+        AND "Transactions"."hostCurrencyFxRate" != 1
+        AND pm.id = "Transactions"."PaymentMethodId"
+        AND c.id = pm."CollectiveId"
+        AND pm."type" = 'host'
+        AND pm."currency" != c."currency"
+    `);
+    console.info(`Fixed ${updateTs.rowCount} transactions`);
+
+    const [, updatePms] = await queryInterface.sequelize.query(`
+      UPDATE "PaymentMethods"
+      SET "currency" = c."currency"
+      FROM "Collectives" c
+      WHERE
+        c.id = "CollectiveId" 
+        AND "PaymentMethods"."type" = 'host'
+        AND "PaymentMethods"."currency" != c."currency"
+    `);
```

Comment:
This is normal in the history of an account to have different currencies, even host account. They will always start with one in USD. Maybe we should add a JOIN on Transactions to make sure we're only updating the bogus one.